api/src/main/java/org/pmiops/workbench/profile/ProfileMapper.java

```diff
@@ -33,29 +34,36 @@ Profile toModel(
       VerifiedInstitutionalAffiliation verifiedInstitutionalAffiliation,
       DbUserTermsOfService latestTermsOfService,
       Double freeTierUsage,
-      Double freeTierDollarQuota);
+      Double freeTierDollarQuota,
+      String accessTierShortNames,
```

Comment:
neither of these are stored in the DbUser table so we have to supply from elsewhere